site/docs/hosting/fly.md

```diff
@@ -21,15 +21,14 @@ You can run your bot using both [webhooks or long polling](../guide/deployment-t
 ::: code-group
 
 ```ts{11} [Deno]
-import { serve } from "https://deno.land/std/http/server.ts";
 import { webhookCallback } from "https://deno.land/x/grammy/mod.ts";
 // You might modify this to the correct way to import your `Bot` object.
 import { bot } from "./bot.ts";
 
 const port = 8000;
 const handleUpdate = webhookCallback(bot, "std/http");
 
-serve(async (req) => {
+Deno.serve(async (req) => {
```

Comment:
The server options is the first argument in `Deno.serve` API. So, that `{ port }` (the second argument here) needs to go up.